File: Python/tests/test_peer_ocu.py
import pytest

from hbayesdm.models import peer_ocu


def test_peer_ocu():
    _ = peer_ocu(
        data="example", niter=10, nwarmup=5, nchain=1, ncore=1)


if __name__ == '__main__':
    pytest.main()

File: R/tests/testthat/test_peer_ocu.R
context("Test peer_ocu")
library(hBayesDM)

test_that("Test peer_ocu", {
  # Do not run this test on CRAN
  skip_on_cran()

  expect_output(peer_ocu(
      data = "example", niter = 10, nwarmup = 5, nchain = 1, ncore = 1))
})
